modules/ROOT/pages/dataweave-formats-flatfile.adoc

```diff
@@ -29,8 +29,116 @@ payload
 
 // CONFIG PROPS GET APPENDED BELOW /////////////////////////////////
 
-// Flat File config properties are shared with COBOL Copybook and Fixed Width.
-include::partial$dataweave-formats-flatfile-config.adoc[]
+
+[[properties]]
+== Configuration Properties
+
+DataWeave supports the following configuration properties for this format.
+
+
+[[reader_properties]]
+=== Reader Properties
+
+This format accepts properties that provide instructions for reading input data.
+
+
+[cols="1,1,1,3a", options="header"]
+|===
+|Parameter |Type |Default|Description
+|`allowLenientWithBinaryNotEndElement`  |`Boolean`|`false`|When the schema contains elements of type Binary or Packed, the `lenient` option does not allow short records, regardless of the last element's
+format type. When you set this property to `true`, the validation applies only to cases in which the record ends with type Binary or Packed.
+
+Valid values are `true` or `false`.
+|`enforceRequires`  |`Boolean`|`false`|Produces an error when set to `true` if a required value is missing.
+
+Valid values are `true` or `false`.
+|`missingValues`  |`String`|`null`|Fill character used to represent missing values. To activate a non-default setting, set the `useMissCharAsDefaultForFill` property to `true`, and use one of the following values to `missingValues`:
```

Comment:
```suggestion
|`missingValues`  |`String`| `nulls` for COPYBOOK, `spaces` otherwise |Fill character used to represent missing values. To activate a non-default setting, set the `useMissCharAsDefaultForFill` property to `true`, and use one of the following values to `missingValues`:
```

Reply:
need this fix in the include

Reply:
this is already in the include